src/config.rs

```diff
@@ -151,15 +151,18 @@ impl Default for RawUpdatesConfig {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
 struct RawDirectoriesConfig {
     #[serde(default)]
-    pub custom_pages_dir: Option<PathBuf>,
+    pub custom_pages_dir: PathBuf,
```

Comment:
I am not really sure if my understanding is correct here, but I thought that all the `Raw*Config`s are just "dumb" data containers, that are "materialized" into meaningful values (with smart defaults and all) when converting to their `*Config` counterparts. Then again, this happened here before :man_shrugging: 

@dbrgn why exactly do we have this distinction again?

Reply:
Yeah I'm not sure either tbh. Kinda reminds me of the DTO pattern, since `RawConfig` is the one that's serializable and `Config` is not.

Reply:
If `PathBuf` is not an `Option`, then deserializing a TOML config file without a `custom_pages_dir` specified will fail.

(Actually, in this case it will use the default value of `PathBuf` due to the annotation, but I'm not sure what that is, and I don't think it's meaningful.)

@niklasmohrin is right, `Raw*Config` is purely here for deserializing the config files into Rust-y values, and then the non-raw config objects contain validated and potentially more structured values.

Reply:
@dmaahs2017 can you revert the removal of the options? otherwise we cannot differentiate between someone not setting the path in the config at all (which is valid of course) and someone setting an empty path (which is invalid).

With your change, if I'm not mistaken, the default value will be used if the custom pages dir is not set by the user (which is an empty path). This empty path will be joined with a filename. This means that a user that does not set the custom pages dir, but has a `tldr.tar` file in the current directory, will be served this local file instead of the actual tar page.